cmake/Dependencies.cmake

```diff
@@ -1320,6 +1310,10 @@ if (NOT BUILD_ATEN_MOBILE)
   SET(AT_MKLDNN_ENABLED 0)
   SET(CAFFE2_USE_MKLDNN OFF)
   IF (USE_MKLDNN)
+    # Anything else this works on?
+    IF (CMAKE_SYSTEM MATCHES "Linux" OR APPLE)
+      hunter_add_package(mkldnn)
+    endif()
     INCLUDE(${CMAKE_CURRENT_LIST_DIR}/public/mkldnn.cmake)
```

Comment:
Are you sure it will work? I don't think upstream code:
- https://github.com/hunter-packages/pytorch/blob/83221655a8237ca80f9673dad06a98d34c43e546/cmake/public/mkldnn.cmake#L1-L6
- https://github.com/hunter-packages/pytorch/blob/83221655a8237ca80f9673dad06a98d34c43e546/cmake/Modules/FindMKLDNN.cmake#L11-L22

Is compatible with CMake API used in Hunter:
- https://docs.hunter.sh/en/latest/packages/pkg/mkldnn.html

If you can't test it probably it's better to remove it.

Reply:
OK, I'll remove it. It compiles without it anyway.